File: csharp/SlimDownYourAggregates.Tests/Slimmed/Events/FormatRemovedEvent.cs
using SlimDownYourAggregates.Tests.Slimmed.Core;
using SlimDownYourAggregates.Tests.Slimmed.Entities;

namespace SlimDownYourAggregates.Tests.Slimmed.Events;

public record FormatRemovedEvent(
    BookId BookId,
    Format Format
): IDomainEvent;

File: java/src/main/java/io/eventdriven/slimdownaggregates/slimmed/events/FormatRemovedEvent.java
package io.eventdriven.slimdownaggregates.slimmed.events;

import io.eventdriven.slimdownaggregates.slimmed.core.IDomainEvent;
import io.eventdriven.slimdownaggregates.slimmed.entities.BookId;
import io.eventdriven.slimdownaggregates.slimmed.entities.Format;

public record FormatRemovedEvent(
  BookId bookId,
  Format format
) implements IDomainEvent {
}
